config/initializers/devise.rb

```diff
@@ -5,38 +5,13 @@
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
   config.secret_key = ENV['DEVISE_SECRET_KEY']
-  # ==> Mailer Configuration
```

Comment:
We never mail our educator users. They can't sign up and can't recover or change their passwords through Devise, because we are using our district partners' LDAP system for authentication.